geomstats/__about__.py

```diff
@@ -0,0 +1,15 @@
+__version__ = '1.11-dev'  # Remove -dev before releasing
+
+install_requires = [
+    'autograd',
+    'numpy>=1.14.1',
+    'scipy',
+    'matplotlib',
+    ]
+
+extras_require = {
+    'test': ['nose2', 'coverage', 'codecov'],
+    'tf': ['tensorflow>=1.8'],
+    'torch': ['torch==0.4.0'],
+    }
+extras_require['all'] = sum(extras_require.values(), [])
```

Comment:
nit - python doesn't recommend summing other things than numbers [1]. Also it makes the code harder to read for people not familiar with the internals of summation of lists (and is one of the least efficient methods for concatenating lists of strings - but it doens't matter here with the size).

Maybe just something like 

`list(itertools.chain(*extras_require.values())) `



[1] https://docs.python.org/3/library/functions.html#sum